_data-prepper/pipelines/configuration/sources/documentdb.md

```diff
@@ -0,0 +1,94 @@
+        sts_role_arn: "arn:aws:iam::123456789012:role/MyRole"
+      s3_bucket: my-bucket
+      s3_region: us-west-2
+      collections:
+        - collection: my-collection
+          export: true
+          stream: true
+      acknowledgments: true
+```
+
+## Configuration
+
+You can use the following options to configure the `documentdb` source.
+
+Option | Required | Type | Description
+:--- | :--- | :--- | :---
+`host` | Yes | String  | The hostname of the DocumentDB cluster.
+`port` | No | Integer | The port number of the DocumentDB cluster. Defaults to `27017`.
+`trust_store_file_path` | No | String | The path to a trust store file that contains the public certificate for the DocumentDB cluster.
+`trust_store_password` | No | String | The password for the trust store specified by `trust_store_file_path`.
+`authentication` | Yes | authentication | The authentication configuration. See [authentication](#authentication) for more information.
+`collections` | Yes | List | A list of [collection](#collection) configurations. Exactly one collection is required.
+`s3_bucket` | Yes | String  | The Amazon S3 bucket to use for processing events from DocumentDB.
+`s3_prefix` | No | String  | An optional key prefix in Amazon S3. By default, there is no key prefix.
+`s3_region` | No | String  | The AWS region where the S3 bucket resides.
+`aws` | Yes | AWS | The AWS configuration. See [aws](#aws) for more information.
+`id_key` | No | String  | When specified, the `_id` field from DocumentDB will be set to the key name specified by `id_key`. You can use this when need more information beyond an ObjectId string saved to your sink. By default, the `_id` is not made as part of the Event.
+`direct_connection` | No | Boolean  | When `true`, the MongoDB driver will connect directly to the specified DocumentDB server or servers without discovering and connecting to the entire replica set. Defaults to `true`.
+`read_preference` | No | String  | Determines how to read from DocumentDB. See [Read Preference Modes](https://www.mongodb.com/docs/v3.6/reference/read-preference/#read-preference-modes) for details. Defaults to `primaryPreferred`.
+`disable_s3_read_for_leader` | No | Boolean  | When `true`, the current leader node will not read from S3. It will only do the work of reading the stream. Defaults to `false`.
+`partition_acknowledgment_timeout` | No | Duration  | Configures how long a node will hold onto a partition. Defaults to `2h`.
+`acknowledgments` | No | Boolean  | When `true`, enables [end-to-end acknowledgments]({{site.url}}{{site.baseurl}}/data-prepper/pipelines/pipelines#end-to-end-acknowledgments) on this source after events are sent to the sinks.
+`insecure` | No | Boolean | Disables TLS. Defaults to `false`. Do not use this value in production.
+`ssl_insecure_disable_verification` | No | Boolean | Disables TLS hostname verification. Defaults to `false`. Do not use this value in production. Use the `trust_store_file_path` to verify the hostname.
```

Comment:
```suggestion
`ssl_insecure_disable_verification` | No | Boolean | Disables TLS hostname verification. Defaults to `false`. Use the `trust_store_file_path` to verify the hostname. Do not use this value in production. 
```

Reply:
@vagimeli , I took a different approach here. I found the clearest way to communicate this was to leave the order as it is. I made some changes in my latest revision which include the word "instead" to try to clarify that one configuration is better than the other.